.github/workflows/node-ci.yml

```diff
@@ -79,6 +79,13 @@ jobs:
         id: changed
       - run: echo '${{ steps.changed.outputs.files }}'
 
+      # Test json-args with a failing command
+      - uses: ./actions/json-args
+        continue-on-error: true
+        with:
+          list: '["hi","folks with space"]'
+          run: node -e 'console.log(process.argv); process.exit(1)' {}
```

Comment:
**suggestion (non-blocking):** I forgot that you were using node-ci.yml for testing.  You may want to rename the workflow file to something like internal-action-tests.yml (or something like that) so that it's more obvious that changes like these are meant to be committed (I initially thought this was temporary test code that should be removed).